### ISSUE TITLE
cheese classification

### ISSUE THREAD

**Issue by adi271001** [abhisheks008/ML-Crate#655]
### ML-Crate Repository (Proposing new issue)
:red_circle: **Project Title** : cheese classification
:red_circle: **Aim** : to classify cheese using various ml algorithms
:red_circle: **Dataset** : https://www.kaggle.com/datasets/jainaru/cheese-across-the-world
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name :  Aditya D
- GitHub Profile Link :  https://www.github.com/adi271001
- Participant ID (If not, then put NA) : NA
- Approach for this Project :  data cleaning , preprocessing , eda , using ml models like knn , logistic regression , decision tree , random forest , svm test and submit
- What is your participant role? (Mention the Open Source Program name. Eg. HRSoC, GSSoC, GSOC etc.) SSOC 

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by why-aditi** [comment]
Full name : Aditi Kala
GitHub Profile Link : https://www.github.com/why-aditi
Participant ID (If not, then put NA) : NA
Approach for this Project : Data cleaning, preprocessing, training DL models both CNN and pretrained ones
What is your participant role? SSOC'24

**Comment by adi271001** [comment]
@abhisheks008 i have done the requested changes on the previous issue please assign me this issue if everything checks out

**Comment by Nithish-456** [comment]
Full name : Nithish Paidimarri
GitHub Profile Link : https://www.github.com/Nithish-456
Participant ID (If not, then put NA) : NA
Approach for this Project : Begin with exploratory data analysis (EDA), Feature engineering , Enhancing the accuracy by hyperparameter tuning (Grid Search ) through different classification algorithms like Random Forest Classifier, Gradient Boosting, XGBoost, SVM etc..
What is your participant role? (SSOC'24)

**Comment by abhisheks008** [comment]
Hi @adi271001 thanks for creating this issue. Can you please share your approach and what are the models you are planning to implement for this problem statement?

**Comment by adi271001** [comment]
Approach for this Project : data cleaning , preprocessing , eda , using ml models like knn , logistic regression , decision tree , random forest , svm  and then test the model

**Comment by abhisheks008** [comment]
> Approach for this Project : data cleaning , preprocessing , eda , using ml models like knn , logistic regression , decision tree , random forest , svm and then test the model

Implement 6-7 models for this problem statement along with the EDA implementations.

Assigned this issue to you @adi271001
